api.go

```diff
@@ -462,6 +462,25 @@ func (cl *Client) GetSearchSuggestions(str string) ([]*SearchSuggestionItem, err
 	return response.Data, nil
 }
 
+func (cl *Client) AddProjectToSelectionFromRepoName(selectionID string, repoName string) (bool, error) {
```

Comment:
New function to both search for a suggestion from a repo name (and get its project key) and add that project to a list.